n2y/notion.py

```diff
@@ -24,6 +24,9 @@
     UseNextClass, is_api_error_code, APIErrorCode
 )
 
+
+# TODO: Rename this file `client.py`
```

Comment:
Is this TODO to be addressed in this PR?

Reply:
No it is not.